yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -228,51 +166,14 @@ func prepareImportDataStatusTable(streamChanges bool) ([]*tableMigStatusOutputRo
 		}
 	}
 
-	snapshotRowCount := make(map[string]int64)
-	if importerRole == FB_DB_IMPORTER_ROLE {
-		exportStatus, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
+	for _, dataFile := range dataFileDescriptor.DataFileList {
```

Comment:
Just confirming: Using this as opposed to `msr.TableListExportedFromSource` because now in import data status, we don't care about changes, only snapshot. Correct?

Reply:
yes correct